--- a/bootstrap
+++ b/bootstrap
@@ -225,7 +225,7 @@
   if test x"$find_tool_res" = x; then
     for i
     do
-      if ($i --version </dev/null) >/dev/null 2>&1; then
+      if ($i --help </dev/null) >/dev/null 2>&1; then
        find_tool_res=$i
        break
       fi
@@ -235,8 +235,8 @@
   fi
   test x"$find_tool_res" != x \
     || die "one of these is required: $find_tool_names"
-  ($find_tool_res --version </dev/null) >/dev/null 2>&1 \
-    || die "${find_tool_error_prefix}cannot run $find_tool_res --version"
+  ($find_tool_res --help </dev/null) >/dev/null 2>&1 \
+    || die "${find_tool_error_prefix}cannot run $find_tool_res --help"
   eval "$find_tool_envvar=\$find_tool_res"
   eval "export $find_tool_envvar"
 }
