--- a/gnulib-tool
+++ b/gnulib-tool
@@ -1660,7 +1660,7 @@
         lkbase=`echo "$lkfile" | sed -e 's,^.*/,,'`
         rm -f "$tmp/$lkbase"
         cp "$gnulib_dir/$lkfile" "$tmp/$lkbase"
-        patch -s "$tmp/$lkbase" < "$func_lookup_local_file_result" >&2 \
+        patch "$tmp/$lkbase" < "$func_lookup_local_file_result" >&2 \
           || func_fatal_error "patch file $func_lookup_local_file_result didn't apply cleanly"
         lookedup_file="$tmp/$lkbase"
         lookedup_tmp=true
